src/go/server/server.go

```diff
@@ -151,8 +153,24 @@
 		panic(err)
 	}
 	accountId := caEvent["accountId"].(string)
+	account := caEvent["account"].(struct {
+		AccountId  [16]uint8      `json:"accountId"`
+		Owner      common.Address `json:"owner"`
+		PublicKeyX *big.Int       `json:"publicKeyX"`
+		PublicKeyY *big.Int       `json:"publicKeyY"`
+		Curve      uint8          `json:"curve"`
+		IsLocked   bool           `json:"isLocked"`
+	})
```

Comment:
In reality, caEvent["account"] is a mapping and holds a value of type interface{}, so it cannot be cast directly to the Account type. Resolving this would require a slightly more complex approach. However, since there are only two instances of this issue, we will leave it as is.